### ISSUE TITLE
tflite can't predict correctly as a keras model h5

### ISSUE THREAD

**Issue by imaneBYD2020** [tensorflow/community#336]
# Community repo

The only issues you should file here concern the community's documentation or processes.

All other bugs should be filed on the appropriate repo, questions directed to the various email groups, or if it's a "how-to" question, StackOverflow.

**Comment by imaneBYD2020** [comment]
Hello i use a model keras saved in h5 format it can predict corectly the classes i convert it into a tflite , it can predictjust one class it fixed
here is my keras model:
base_model = NASNetMobile(include_top=False, weights='imagenet',input_shape=(image_size, image_size, 3)) 
bn = BatchNormalization()(inputs)
x = base_model(bn)
x = GlobalAveragePooling2D()(x)
x = Dense(512, activation='relu')(x)
x = Dropout(0.5)(x)
x = Dense(512, activation='relu')(x)
x = Dropout(0.5)(x)
#kernel_initializer='uniform'
output = Dense(len(classes), activation='softmax')(x)
model = Model(inputs, output)
model.compile(optimizer=Adam(0.001), loss='categorical_crossentropy', metrics=['acc'])



and here is the code to convert model to tflite:

import tensorflow.lite as lite
keras_model_path = "mymodel.h5"
tflite_model_path = "mymodel.tflite"
model.save(keras_model_path)
converter = tf.compat.v1.lite.TFLiteConverter.from_keras_model_file(keras_model_path)
tflite_model = converter.convert()
open(tflite_model_path, "wb").write(tflite_model)

**Comment by mihaimaruseac** [comment]
This is not the right place for this issue.

> The only issues you should file here concern the community's documentation or processes.

Please file an issue in https://github.com/tensorflow/tensorflow
